app/views/govuk_component/task_list_related.raw.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  links ||= false
+  title ||= "Part of"
+%>
+<% if links %>
+  <%
+    one_link = links.length == 1 ? true : false
+  %>
+  <div class="pub-c-task-list-related">
+    <h2 class="pub-c-task-list-related__heading">
+      <span class="pub-c-task-list-related__pretitle"><%= title %></span>
+
+      <% if one_link %>
+          <a href="<%= links[0][:href] %>" class="pub-c-task-list-related__link-item">
+            <%= links[0][:text] %>
+          </a>
+        </h2>
+      <% else %>
+        </h2>
+        <ul class="pub-c-task-list-related__links">
+          <% links.each do |link| %>
+            <li class="pub-c-task-list-related__link">
```

Comment:
Should the `pub-c-task-list-related__link` and `pub-c-task-list-related__link-item` classes be the other way around? I.e: the `pub-c-task-list-related__link` class should be on the `<a>` tag

Reply:
Updated, good point.